rootston/seat.c

```diff
@@ -14,6 +15,8 @@
 static void handle_keyboard_key(struct wl_listener *listener, void *data) {
 	struct roots_keyboard *keyboard =
 		wl_container_of(listener, keyboard, keyboard_key);
+	struct roots_desktop *desktop = keyboard->input->server->desktop;
+	wl_signal_emit(&desktop->idle->activity_notify, NULL);
```

Comment:
Why don't you call `wlr_idle_notify_activity` in rootston?

Reply:
where else in rootston I could call this?

Reply:
No, the place from which you notify activity (ie. adding a line here) is fine. But you're triggering the event manually. Can you call `wlr_idle_notify_activity` instead? (`wlr_idle_notify_activity` is unused otherwise)

Reply:
You are totally right. My bad. Sorry about this.